resources/js/Pages/Results.vue

```diff
@@ -79,15 +79,27 @@
                         :disabled="!user"
                         @decision="recordDecision"
                     />
-                    <div class="form-buttons">
-                        <wikit-button
-                            :disabled="!user"
-                            variant="primary"
-                            type="progressive"
-                            native-type="submit"
-                        >
-                            {{ $i18n('result-form-submit') }}
-                        </wikit-button>
+                    <div class="form-footer">
+                        <div class="form-success-message">
```

Comment:
No, the message only appears after clicking the button, but I need an empty `div` to always be there, so that the buttons remain at the end of the line. Happy to discuss alternatives.

Reply:
There are several alternative ways of doing this, without having to "reserve the space" so to speak.

Reply:
OK, thanks for your help on this. The space-reserving div is gone and replaced by proper css rules.